images/regen-sandbox/README.md

```diff
@@ -0,0 +1,31 @@
+# Regen Ledger Sandbox
+
+This container (and corresponding setup scripts) is intended to aid with end-to-end testing of client libraries, applications, manual local network tests, and other scenarios where you want to be able to quickly bootstrap a single node regen network (targeting any version or commit hash of regen-ledger), and populate the network with some basic data for your end-to-end testing or exploratory purposes.
+
+
+Build locally:
+```sh
+docker build . -f images/regen-sandbox/Dockerfile -t regen-sandbox
+```
+
+Run the container:
+```sh
+# Optionally pass your own testing memonic.
+# If no $REGEN_MNEMONIC is provided, a mnemonic
+# will be auto-generated and printed to STDOUT
+export REGEN_MNEMONIC="YOUR TESTING MNEMONIC"
+
+(cd images/regen-sandbox && docker run -v $(pwd):/regen --env REGEN_MNEMONIC regen-sandbox:latest)
+```
+
+The above command will start up a new chain from genesis, with 5 accounts [`addr1`, ...`addr5`]. All accounts are generated from the same mnemonic with incremental `--account` indices (using HD derivation), and seeded with 10000 REGEN tokens. `addr1` is set as the single validator in this network and has an additional 40000 REGEN tokens, all of which are self-delegated.
+
+If a `./.regen` home directory is detected, it will not initiatize a new chain, but simply run `regen start` with the existing home directory. If you want to always create a fresh home directory, you can override this behavior by adding a `-o` or `--overwrite-home-dir` flag to the end of your `docker run` command.
+
+You can additionally provide a comma separated list of setup scripts to run as an argument at the end of the `docker run` command, like so:
+
+```
+docker run -v $(pwd):/regen --env REGEN_MNEMONIC regen-sandbox:latest ecocredit,data
+```
+
+This will look in `./setup` for two files, `./setup/ecocredit.sh` and `./setup/data.sh` and run them in order. Stopping the docker process and running with these arguments a second time will attempt to run the scripts again and potentially popupate a second round of data.
```

Comment:
This is resolved now.  It was also more than just a logging problem. The scripts were hardcoded to always add projects / batches / bridges to C01 (assuming that it didn't already exist).

I've now changed it so that each script queries the class-id generated and uses that downstream for the rest of the transactions.

Fixed in [68c312f](https://github.com/regen-network/regen-ledger/pull/1550/commits/68c312f6a4d30e3e5bc769b1a1d02a7eb0ba26be)